examples/client_test.cpp

```diff
@@ -1696,14 +1696,17 @@ MAGNETURL is a magnet link
 			{
 				for (announce_entry const& ae : h.trackers())
 				{
+					auto best_ae = std::min_element(ae.endpoints.begin(), ae.endpoints.end()
```

Comment:
afaict, this prints the first endpoint with the lowest fail counter. I imagine in the future it might be reasonable to print all of them, right?